features/infrastructure/hosting-jitsi.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Jitsi Hosting
```

Comment:
@user512 / @cjerozal / @maximegalon5  - I'm trying to use feature-driven development so that it's possible to focus on defining what we want to accomplish at the appropriate level of specificity so folks that aren't particularly code-y can riff on it. It also could serve as a reasonable source of usage documentation on a feature-by-feature basis; even if we don't wind up automating them.

I'm curious if this feels like a useful way for us to communicate more asynchronously about intent?

Reply:
I think this is useful.